public/assets/img/RIFOS RNS MANAGER.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<!-- saved from url=(0073)file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/admin-resolution-qr.html -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+      
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <meta http-equiv="x-ua-compatible" content="ie=edge">
+      <meta name="description" content="RIFOS RNS MANAGER">
+      <meta name="keywords" content="">
+      <meta name="author" content="RIFOS">
+      <title>RIFOS RNS MANAGER</title>
+      <!-- favicons -->
+      <link rel="apple-touch-icon" sizes="180x180" href="file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/assets/favicons/apple-touch-icon.png">
+      <link rel="icon" type="image/png" sizes="32x32" href="file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/assets/favicons/favicon-32x32.png">
+      <link rel="icon" type="image/png" sizes="16x16" href="file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/assets/favicons/favicon-16x16.png">
+      <link rel="manifest" href="file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/assets/favicons/site.webmanifest">
```

Comment:
This `href` value is unlikely to work, when run externally.

Is this file necessary for the faucet?